src/content/en/updates/2017/03/chrome-58-media-updates.md

```diff
@@ -0,0 +1,193 @@
+
+# Media Updates in Chrome 58 {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+- Developers can now [customize media controls](#controlslist) such as the
+  download, fullscreen and remoteplayback buttons.
+- Sites installed using the "Add to Homescreen" flow can [autoplay audio
+  and video in the manifest's scope](#autoplay).
+- Chrome on Android now [pauses autoplaying muted video when invisible](#offscreen).
+- Developers can now access the approximate range of colors supported by Chrome and
+  output devices using the [`color-gamut` Media Query](#colorgamut).
+
+## Media controls customization {: #controlslist}
+
+Developers can now customize Chrome's native media controls such as the
+download, fullscreen and [remoteplayback] buttons using the new [ControlsList API].
+
+<div class="attempt-right">
+<figure>
+  <img src="/web/updates/images/2017/03/media-controls.png"
+       alt="Native media controls in Chrome 58">
+  <figcaption>
+    <b>Figure 1.</b>
+    Native media controls in Chrome 58
+  </figcaption>
+</figure>
+</div>
+
+This API aims to offer a way to control the native controls elements/buttons
+that are being shown by the browser to remove some features that do not make
+sense or are not part of the expected user experience, or only whitelist a
+limited amount of features.
+
+The current implementation for now is a blacklist mechanism on native controls
+with the ability to set them directly from HTML content using the new 
+attribute `controlsList`. Check out the [official
+sample](https://googlechrome.github.io/samples/media/controlslist.html).
+
+<div class="clearfix"></div>
+
+Usage in HTML:
+
+<pre class="prettyprint lang-html">
+&lt;video controls <b>controlsList="nofullscreen nodownload noremote foobar"</b>>&lt;/video>
+</pre>
+
+Usage in Javascript:
+
+    var video = document.querySelector('video');
+    video.controls; // true
+    video.controlsList; // "nofullscreen nodownload noremote" - "foobar" not present
+    video.controlsList.remove('noremote');
+    video.controlsList; // "nofullscreen nodownload" - "noremote" not present
+    video.getAttribute('controlsList'); // "nofullscreen nodownload"
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/tFuQd3AcsIQ/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/features/5737006365671424) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=685018)
+
+## Autoplay for Progressive Web Apps added to Home screen {: #autoplay }
+
+Previously, Chrome used to block all `autoplay` with sound on Android without
+exception. This is no longer true. From now on, sites installed using the
+[improved Add to Home screen] flow are allowed to autoplay audio and video
+served from origins included in the [manifest's scope] without restrictions.
```

Comment:
Addressed in https://github.com/google/WebFundamentals/pull/4335/commits/a8335286c2854bf3ced2d25b3ce39c950e67ee52